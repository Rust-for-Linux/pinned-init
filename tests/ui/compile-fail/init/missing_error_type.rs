use pin_init::*;

struct Foo {
    x: Box<usize>,
}

fn main() {
    let _ = try_init!(Foo { x: Box::new(0)? }?);
}
