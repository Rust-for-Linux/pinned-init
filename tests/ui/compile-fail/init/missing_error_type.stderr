error: unexpected end of input, expected one of: `for`, parentheses, `fn`, `unsafe`, `extern`, identifier, `::`, `<`, `dyn`, square brackets, `*`, `&`, `!`, `impl`, `_`, lifetime
 --> tests/ui/compile-fail/init/missing_error_type.rs:8:13
  |
8 |     let _ = try_init!(Foo { x: Box::new(0)? }?);
  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `try_init` (in Nightly builds, run with -Z macro-backtrace for more info)
